kodilla-intro/src/main/java/com/kodilla/task25/User.java

```diff
@@ -0,0 +1,37 @@
+package com.kodilla.task25;
+
+public class User {
+    final String name;
+    final int age;
+
+    public User(String name, int age) {
+        this.name = name;
+        this.age = age;
+    }
+
+    public static void main(String[] args) {
+        User maria = new User("Maria", 30);
+        User ewa = new User("Ewa", 57);
+        User scooby = new User("Scooby", 4);
+        User henryk = new User("Henryk", 32);
+
+        User[] users = {maria, ewa, scooby, henryk};
+
+        int result = 0;
+        double average;
+        int userNumber = users.length;
+        for (User user : users) {
+            result = result + user.age;
+        }
+        average = result / users.length;
+
+        for (User user : users) {
+            if (user.age < average) {
+                System.out.println(user.name);
+            } else {
+                System.out.println("Nobody");
+            }
+        }
+    }
+}
```

Comment:
napisz odrębną metody:
- `private static User[] init()` - inicjującą tablicę userów (to co jest w liniach 13-18 de facto)
- `private static double average(User[] users)` zwracająca średnią wieku wszystkich użytkowników
- `private static void printYoungerThan(User[] users, double average)` - drukującą użytkowników, których wiek jest poniżej podanej jako parametr średniej wieku

Metoda main powinna wyglądać tak:
`private static void main(String[] args) {
User[] users = init();
double average = average(users);
print(users, average);`